src/components/application_manager/test/policy_handler_test.cc

```diff
@@ -1387,6 +1387,22 @@ TEST_F(PolicyHandlerTest, IsRequestTypeAllowed) {
   policy_handler_.IsRequestTypeAllowed(kPolicyAppId_, type);
 }
 
+TEST_F(PolicyHandlerTest, IsRequestSubTypeAllowed) {
+  // Arrange
+  EnablePolicyAndPolicyManagerMock();
+
+  // Check expectations
+  EXPECT_CALL(*mock_policy_manager_, GetAppRequestSubTypesState(kPolicyAppId_))
+      .WillOnce(Return(policy::RequestSubType::State::AVAILABLE));
+  EXPECT_CALL(*mock_policy_manager_, GetAppRequestSubTypes(kPolicyAppId_))
+      .WillOnce(
+          Return(std::vector<std::string>({"fakeSubType", "fakeSubType2"})));
```

Comment:
@BSolonenko below you're define "fakeSubType" as variable, please define it above and use it
As well remove comments